ios/rntemplate.xcodeproj/project.pbxproj

```diff
@@ -798,7 +798,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = i386;
```

Comment:
we should keep this as it was to avoid compatibility issues with Apple Silicon Macs (M1)